.github/linters/ct.yaml

```diff
@@ -7,3 +7,4 @@ chart-dirs:
 excluded-charts:
 chart-repos:
 validate-chart-schema: false
+check-version-increment: false
```

Comment:
Disabling this for now... not sure the chart version should be bumped, even if the image is changed, until we make a release.